src/python/zquantum/qaoa/ansatzes/x_ansatz.py

```diff
@@ -0,0 +1,117 @@
+from zquantum.core.interfaces.ansatz import Ansatz, ansatz_property
+
+from zquantum.core.circuits import Circuit
+from zquantum.core.evolution import time_evolution
+from zquantum.core.openfermion import change_operator_type
+
+from openfermion import QubitOperator, IsingOperator
+from openfermion.utils import count_qubits
+from typing import Tuple, Union, Optional, List
+import numpy as np
+import sympy
+from overrides import overrides
+from itertools import combinations
+
+
+class QAOAXAnsatz(Ansatz):
+
+    supports_parametrized_circuits = True
+    cost_hamiltonian = ansatz_property("cost_hamiltonian")
```

Comment:
This ansatz does not require cost Hamiltonian to be present, I think this field can be removed.